src/pages/studyView/virtualStudy/VirtualStudy.tsx

```diff
@@ -89,16 +89,10 @@ export default class VirtualStudy extends React.Component<IVirtualStudyProps, {}
                     return acc;
                 }, []);
 
-                let filters = { patients: {}, samples: {} };
-
-                /* 
-                    TODO: this is to support existing virtual study feature.
-                    but eventually we need to save StudyViewFilter
-                 */
                 let parameters = {
                     name: this.name || this.namePlaceHolder,
                     description: this.description,
-                    filters: filters,
+                    studyViewFilter: _.omit(this.props.filter, ['sampleIdentifiersSet']),
```

Comment:
What about the custom charts?

Reply:
Not sure. these are actually not captured in `studyViewFilter` class in backend. however I assume `sampleIdentifiers` would be updated accordingly for those charts

Reply:
is sampleIdentifiersSet a type property?  If so, it would be best to avoid the above because putting property name as string is not typesafe.